[SSL for SaaS] add using dashboard for SSL for SaaS and change string processing in uploading_certificates.md (#1457)
cloudflare/cloudflare-docs: products/ssl/src/content/ssl-for-saas/getting-started.md

--------

## Configure zone with proxy fallback origin

> The fallback origin is where the traffic of your Custom Hostnames will be routed to.  The fallback record is the value of the DNS record you set up via step 2 below.

Reviewer: ```suggestion
The fallback origin is where the traffic of your Custom Hostnames will be routed to.  The fallback record is the value of the DNS record you set up via step 2.
```

1. Sign up your desired zone at [www.cloudflare.com](https://www.cloudflare.com) and select the Free plan.
2. Under the **DNS** tab on the dashboard, add a DNS record pointing to the origin IP address (fallback origin) for Cloudflare to send custom hostname traffic. This hostname is not provided to customers.  Name the hostname according to your naming convention.  For example: `proxy-fallback.saasprovider.com`.

> 3. Set up an additional DNS CNAME record to serve as the CNAME target for your end customers.  CNAME this record to your fallback origin, and use a more user-friendly CNAME target for customers.  This record can optionally be a wildcard, e.g. `*.customers.saasprovider.com`.

Reviewer: ```suggestion
3. Set up an additional DNS CNAME record to serve as the CNAME target for your end customers. CNAME this record to your fallback origin and use a more user-friendly CNAME target for customers. This record can optionally be a wildcard, e.g. `*.customers.saasprovider.com`.
```

--------

## Configure zone with proxy fallback origin

> The fallback origin is where the traffic of your Custom Hostnames will be routed to.  The fallback record is the value of the DNS record you set up via step 2.

Reviewer: ```suggestion
The fallback origin is where the traffic of your Custom Hostnames will be routed to. The fallback record is the value of the DNS record you set up via step 2.
```

1. Sign up your desired zone at [www.cloudflare.com](https://www.cloudflare.com) and select the Free plan.
1. Click **DNS**.

> 1. Add a DNS record pointing to the origin IP address (fallback origin) for Cloudflare to send custom hostname traffic. This hostname is not provided to customers.  Name the hostname according to your naming convention.  For example: `proxy-fallback.saasprovider.com`.

Reviewer: ```suggestion
1. Add a DNS record pointing to the origin IP address (fallback origin) for Cloudflare to send custom hostname traffic. This hostname is not provided to customers. Name the hostname according to your naming convention. For example: `proxy-fallback.saasprovider.com`.
```

```bash
$ curl -XPOST "https://api.cloudflare.com/client/v4/zones/:zone_id/custom_hostnames"\
       -H "X-Auth-Email: {email}" -H "X-Auth-Key: {key}"\
       -H "Content-Type: application/json"\
       -d '{"hostname":"app.customer.com", "ssl":{"method":"http","type":"dv"}}'
```

Note that it’s possible to serve these HTTP records from your own web servers, in advance of placing the CNAME. The payload returned includes the path where the CA will look for the challenge along with the body that should be returned.

Alternatively, you may also issue certificates to custom hostnames via the dashboard. 
1. Navigate to **SSL/TLS** > **Custom Hostnames** and click **Add Custom Hostname**.
1. Add your customer's hostname `app.customer.com` and set the relevant options. Click **Add Custom Hostname**.
![Add a custom hostname](..//static/ssl-for-saas-custom-hostname.png)

> 1. You will be brought back to the previous screen, which will show “Pending” before it changes to “Active” after a while. If you see an error stating “custom hostname does not CNAME to this zone”, you need to [set the DNS record at the customer's domain](#setting-cname-at-customer-domain).

Author: ```suggestion
1. You will be brought back to the previous screen, which will show “Pending” before it changes to “Active” within 5 minutes. If you see an error stating “custom hostname does not CNAME to this zone”, you need to [set the DNS record at the customer's domain](#setting-cname-at-customer-domain).
```
I've changed it to within 5 minutes.

Once domain validation has been completed, the certificates will be issued and distributed to Cloudflare’s edge. With a CNAME in place, the entire process—from validation to issuance to edge deployment—completes in approximately 90 seconds.

--------

## Setting CNAME at customer domain

> Your customer needs to set up a CNAME record at their DNS provider, pointing to your CNAME target configured in step 3 [above](#configure-zone-with-proxy-fallback-origin). For example:

Reviewer: ```suggestion
Your customer needs to set up a CNAME record at their DNS provider, pointing to your CNAME target configured in [a previous step](#configure-zone-with-proxy-fallback-origin). For example:
```
